src/test/apps.test.js

```diff
@@ -1090,3 +1091,69 @@ describe("App Status component", function () {
   });
 
 });
+
+describe("Breadcrumb Component", function () {
+  before(function () {
+    this.renderComponent = (group, app, task) => {
+      var renderer = TestUtils.createRenderer();
+      renderer.render(
+        <BreadcrumbComponent group={group} app={app} task={task} />
+      );
+      return renderer.getRenderOutput();
+    };
+  });
```

Comment:
I wouldn't have thought it was necessary - am I missing something obvious? (possibly insufficient Einbecker Bock ;))